recipes/fmt/all/conanfile.py

```diff
@@ -64,65 +58,63 @@ def config_options(self):
             self.options.with_os_api = False
 
     def configure(self):
-        if self.options.header_only:
-            del self.options.fPIC
-            del self.options.shared
-            del self.options.with_os_api
-        elif self.options.shared:
-            del self.options.fPIC
-
-    def validate(self):
-        if self.options.get_safe("shared") and self._is_msvc and "MT" in msvc_runtime_flag(self):
-            raise ConanInvalidConfiguration(
-                "Visual Studio build for shared library with MT runtime is not supported"
-            )
+        try:
+            if self.options.header_only:
+                del self.options.fPIC
+                del self.options.shared
+                del self.options.with_os_api
+            elif self.options.shared:
+                del self.options.fPIC
+        except Exception:
```

Comment:
I think this try: except: is not necessary any more

Reply:
Yes, sorry, it's necessary I was wrong

Reply:
Sorry, you are right, these are options. Only needed to protect a setting (or subsetting) that might not exist in some configuration.